controllers/news.controller.js

```diff
@@ -85,4 +86,19 @@ exports.patchArticleController = (req, res, next) => {
     .catch((err) => {
       next(err)
     })
+}
+
+exports.deleteCommentByIdController = (req, res, next) => {
+  const {comment_id} = req.params
+
+  deleteCommentByIdModel(comment_id)
+    .then(rowCount => {
+      if(!rowCount) {
+        return res.status(404).send({ msg: 'Comment does not exist' });
+      }
+      res.status(204).send();
```

Comment:
A nice alternative to this is `res.sendStatus(204)` which then doesnt need the empty .send at the end